todo_app/tests/test_IntUnit.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from dotenv import load_dotenv, find_dotenv
+from unittest.mock import patch
+import requests
+import os
+from threading import Thread
+from todo_app import app    
+
+@pytest.fixture(scope='module')
+def test_app():
+       application = app.create_app()
+       thread = Thread(target=lambda: application.run(use_reloader=False))
+       thread.daemon = True
+       thread.start()
+       yield app 
+       thread.join(1)
+
+@pytest.fixture
+def client():
+       file_path = find_dotenv('.env.test')
+       load_dotenv(file_path, override=True)
+       test_app = app.create_app()
+       with test_app.test_client() as client:
+          yield client
+
+def get_requests():
+   mock_get_requests = [{"id": "5fb55f9084036928139db350", "title": "Testing", "status": "To Do", "dateLastActivity": "2020-11-18T18:43:33.434Z"}]
+   return mock_get_requests
+
+@patch('requests.get')
+def test_index(client):
+   response = client.get('/')
```

Comment:
This test feels a little confused, currently this hits our index page then checks that the `get_requests` method defined just above returns an item with a title

It might be worth taking a little look at [this article](https://realpython.com/python-mock-library/) under "Patch as a Decorator" to see how we configure the mock; we'll then want to check that the mocked item ends up on the page we loaded (i.e. is in the response), rather than testing the mocked function directly.